tests/unit/tls_test.ts

```diff
@@ -1589,41 +1588,22 @@ Deno.test({
       p1 = listener1.accept().then((conn) => {
         conn.close();
         listener1Recv = true;
+        listener1.close();
```

Comment:
This was assuming that connections are always round-robin'd, but that's not a guarantee.